ui/raidboss/data/06-ew/raid/p1s.ts

```diff
@@ -0,0 +1,204 @@
+import Conditions from '../../../../../resources/conditions';
+import NetRegexes from '../../../../../resources/netregexes';
+import { Responses } from '../../../../../resources/responses';
+// import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export type Data = RaidbossData;
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: 0x3EB,
+  timelineFile: 'p1s.txt',
+  timelineTriggers: [
+    {
+      id: 'P1S Tile Positions',
+      regex: /(?:First|Second|Third) Element/,
+      beforeSeconds: 3,
+      infoText: (_data, _matches, output) => output.positions!(),
+      outputStrings: {
+        positions: {
+          en: 'Tile Positions',
+        },
+      },
+    },
+  ],
+  triggers: [
+    {
+      id: 'P1S Warder\'s Wrath',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '662A', source: 'Erichthonios', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '662A', source: 'Erichthonios', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '662A', source: 'Érichthonios', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '662A', source: 'エリクトニオス', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'P1S Shining Cells',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6616', source: 'Erichthonios', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '6616', source: 'Erichthonios', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '6616', source: 'Érichthonios', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '6616', source: 'エリクトニオス', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'P1S Slam Shut',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6617', source: 'Erichthonios', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '6617', source: 'Erichthonios', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '6617', source: 'Érichthonios', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '6617', source: 'エリクトニオス', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'P1S Gaoler\'s Flail Right => Left',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '65F6', source: 'Erichthonios', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '65F6', source: 'Erichthonios', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '65F6', source: 'Érichthonios', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '65F6', source: 'エリクトニオス', capture: false }),
+      alertText: (_data, _matches, output) => output.directions!(),
+      outputStrings: {
+        directions: {
+          en: 'Right => Left',
```

Comment:
Is it possible to use `Outputs.right` and `Outputs.left` here and them combine with some en: '${first} => ${second}`, so that they don't have to be translated separately?